components/resources/brave_components_strings.grd

```diff
@@ -955,6 +955,16 @@
       >
         Hide sponsored images
       </message>
+      <!-- Rewards BAP Deprecation Notice -->
+      <message name="IDS_REWARDS_BAP_DEPRECATION_HEADER" desc="">
+        Brave Rewards in Japan is switching to a new system!
+      </message>
+      <message name="IDS_REWARDS_BAP_DEPRECATION_ALERT_TEXT" desc="">
+        The tipping feature will be unavailable from March 13th until April 13th. All BAPs will
+        be discontinued by April 13th. Please use your BAP by March 13th. Any BAP remaining in
+        your Brave browser after March 13th cannot be held or transferred by you or Brave, and
+        will be lost. A new and improved system awaits!
```

Comment:
Matching text in comment: https://github.com/brave/internal/issues/747#issuecomment-764774254 should there be a new line between `will be lost.` and `A new and improved system awaits!`? If so, might need to be broken into two parts

Reply:
The text at https://www.figma.com/file/9Pn7htrOeZsbOJ2NcxtxvR/bitFlyer-Promo?node-id=206%3A0 does not have a line break - @codybanxs can you confirm that the text layout in the design is what we want?

Reply:
oh yeah - good call (with Figma link). I guess either works - maybe we can add test plan after which QA can look at, depending on which is the one they should compare against 🙂

Reply:
While the line break works okay on the mobile UI, it does not work well on the desktop oddly enough. Basically I did not put in a line break due to the length and flow of the copy across the board and to keep things visually consistent. If we do want to proceed with that I would need to tweak the container sizing abit which is not a problem.